client/me/account-close/main.jsx

```diff
@@ -146,39 +151,48 @@ class AccountSettingsClose extends Component {
 							<Fragment>
 								<p className="account-close__body-copy">
 									{ translate(
-										'Account closure cannot be undone and will remove all sites and content.'
+										'Account closure cannot be undone. It will remove your account along with all your sites and all their content.'
+									) }
+								</p>
+								<p className="account-close__body-copy">
+									{ translate(
+										'You will not be able to open a new WordPress.com account using the same email address for 30 days.'
 									) }
 								</p>
 								<p className="account-close__body-copy">
 									{ translate(
-										"If you're unsure about what account closure means or have any other questions, " +
-											'{{a}}chat with someone from our support team{{/a}} before going ahead.',
+										'If you have any questions at all about what happens when you close an account, ' +
+											'please {{a}}chat with someone from our support team{{/a}} first. ' +
+											"They'll explain the ramifications and help you explore alternatives. ",
 										{
 											components: {
 												a: <ActionPanelLink href="/help/contact" />,
 											},
 										}
 									) }
 								</p>
+								<p className="account-close__body-copy">
+									{ translate( 'When you\'re ready to proceed, use the "Close account" button.' ) }
+								</p>
 							</Fragment>
 						) }
 					</ActionPanelBody>
 					<ActionPanelFooter>
 						{ ( isLoading || isDeletePossible ) && (
 							<Button scary onClick={ this.handleDeleteClick }>
 								<Gridicon icon="trash" />
-								{ translate( 'Close Account' ) }
+								{ translate( 'Close account' ) }
 							</Button>
 						) }
 						{ hasAtomicSites && (
 							<Button primary href="/help/contact">
-								{ translate( 'Contact Support' ) }
+								{ translate( 'Contact support' ) }
 							</Button>
 						) }
 						{ hasPurchases &&
 							! hasAtomicSites && (
 								<Button primary href="/me/purchases">
-									{ translate( 'Manage Purchases' ) }
+									{ translate( 'Manage purchases' ) }
```

Comment:
:disappointed: 53 existing translations will be lost with this change.
If the string must change, the following option has already been translated **31** times:
`translate( 'Manage Purchases', { context: 'button label'} )` *ES Score: 16*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/account-closure/add-email-hold-details)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).